### PR TITLE
boards: fanke: Add support for FK750M1-VBT6 board

### PR DESCRIPTION
**Pull request by NU-LL**
Adds the device trees, Kconfig, and documentation files.

The following features have been confirmed working on hardware:
* LED
* SPI LCD
* UART
* W25Q64 SPI FLASH

### CODE REVIEW

File: boards/fanke/fk750m1_vbt6/doc/img/fk750m1_vbt6.jpg
Comment (nordicjm) [inline boards/fanke/fk750m1_vbt6/doc/img/fk750m1_vbt6.jpg:1]: Convert to webp, then put through https://tinypng.com/

**Review by nordicjm — COMMENTED** [review]
Remove arm from commit title and replace with fanke

**Comment by NU-LL** [comment]
Hi @nordicjm , my patch is blocked by ci, this looks like it needs to be approved, can you help me see who I should approach for approval? I didn't find any guidance notes about this on the official [contribute-guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#contribute-guidelines)

**Comment by NU-LL** [comment]
I updated the patch but the ci is blocked again, can you help with approval again? @nordicjm

**Comment by NU-LL** [comment]
The ci is failing again, showing that the email formatting in patch 46c6eec963274f7e38d46730d872aea84f51335e is incorrect, but I see that the last patch I forced to push was cd30d2364fce43dc00792ad41710cc38e2beed45, where the formatting has been changed as required. @nordicjm can you help to see what the problem is? thanks

**Comment by kartben** [comment]
You need to update your *author* email, as it's still non compliant

```
git commit --amend --author="Author Name <email@address.com>"
```

**Comment by NU-LL** [comment]
Thanks a lot, I've updated it

**Review by erwango — DISMISSED** [review]

**Comment by NU-LL** [comment]
Someone help me see why ci failed again, I didn't change anything!

**Comment by kartben** [comment]
> Someone help me see why ci failed again, I didn't change anything!

I just re-triggered CI - all good :)

**Review by erwango — APPROVED** [review]

**Review by nordicjm — CHANGES_REQUESTED** [review]

**Comment by NU-LL** [comment]
@nordicjm This just seems like the documentation is still a bit irregular. I've lost my original patch (I tried to sync the latest code in my repository, but that resulted in an extra merge patch), is it possible to merge in this patch first, and then I'll submit a patch to change the documentation?

**Comment by nordicjm** [comment]
> @nordicjm This just seems like the documentation is still a bit irregular. I've lost my original patch (I tried to sync the latest code in my repository, but that resulted in an extra merge patch), is it possible to merge in this patch first, and then I'll submit a patch to change the documentation?

No it needs to be done as one. You need to remove the merge commit, if you have zephyr checked out somewhere do something like this:
```
cd zephyr
git remote add mine git@github.com:NU-LL/zephyr.git
git fetch mine main
git pull
git branch myboardaddition
git checkout myboardaddition
git cherry-pick d0145f6cd5f44be491f34e752a7aef5af65043d8
git push mine
```

You should then have a new branch which you can open a PR against, though make the changes before submitting it. A new branch name will be required anyway since you have used main here and there is a PR job that fails if you use main

**Comment by NU-LL** [comment]
> > @nordicjm This just seems like the documentation is still a bit irregular. I've lost my original patch (I tried to sync the latest code in my repository, but that resulted in an extra merge patch), is it possible to merge in this patch first, and then I'll submit a patch to change the documentation?
> 
> No it needs to be done as one. You need to remove the merge commit, if you have zephyr checked out somewhere do something like this:
> 
> ```
> cd zephyr
> git remote add mine git@github.com:NU-LL/zephyr.git
> git fetch mine main
> git pull
> git branch myboardaddition
> git checkout myboardaddition
> git cherry-pick d0145f6cd5f44be491f34e752a7aef5af65043d8
> git push mine
> ```
> 
> You should then have a new branch which you can open a PR against, though make the changes before submitting it. A new branch name will be required anyway since you have used main here and there is a PR job that fails if you use main

Thanks, I've fixed the docs and resubmitted the pr: https://github.com/zephyrproject-rtos/zephyr/pull/77717

**Pull request closed without merging**